pkg/streamingpromql/testdata/ours-only/native_histograms.test

```diff
@@ -13,3 +13,9 @@ eval range from 0 to 5m step 1m clamp_min(mixed_metric, 2)
 
 eval range from 0 to 5m step 1m clamp_max(mixed_metric, 2)
 	{} _ 1 2 2
+
+# round ignores any histograms
+# Prometheus currently returns min instead of no value as per the documentation
```

Comment:
What does 'min' refer to here?

Reply:
A typo.

Reply:
Looks like the same typo is earlier in the file as well, above the tests for `clamp`